dm/config/task.go

```diff
@@ -785,7 +789,7 @@ func FromSubTaskConfigs(stCfgs ...*SubTaskConfig) *TaskConfig {
 	c.HeartbeatReportInterval = stCfg0.HeartbeatReportInterval
 	c.CaseSensitive = stCfg0.CaseSensitive
 	c.TargetDB = &stCfg0.To // just ref
-	c.OnlineDDLScheme = stCfg0.OnlineDDLScheme
+	c.OnlineDDL = stCfg0.OnlineDDL
```

Comment:
@GMHDBJD https://github.com/pingcap/dm/pull/1877

what to do when downgrade 🤔

Reply:
if the config is from lower version, c.OnlineDDLScheme must be set, so it should be compatible.

So @okJiang  Please keep the original `c.OnlineDDLScheme = stCfg0.OnlineDDLScheme` from backwark compatibility

Reply:
+1

Reply:
+1